src/content/configuration/watch.md

```diff
@@ -70,3 +70,46 @@ poll: 1000 // Check for changes every second
 
 T> If watching does not work for you, try out this option. Watching does not work with NFS and machines in VirtualBox.
 
+# Troubleshooting
+
+## Webpack doesn't recompile on change while watching
+
+### File changes are being seen, just no files are being updated
+
+Verify that webpack is not being notified of changes by running with the --progress flag. If progress shows on save but no files are output, it is likely a configuration issue, not a file watching issue.
+
+```bash
+webpack --watch --progress
+```
+
+### Not enough watchers
+
+Verify that you have enough available watchers in your system. If this value is too low, the file watcher in Webpack won't recognize the changes:
+
+```bash
+cat /proc/sys/fs/inotify/max_user_watches
+```
+
+Arch users, add `fs.inotify.max_user_watches=524288` to `/etc/sysctl.d/99-sysctl.conf` and then execute `sysctl --system`. Ubuntu users (and possibly others): `echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p`.
+
+### macOS fsevents bug
+
+On macOS folders can get corrupted. See this article:
+
+[OS X FSEvents bug may prevent monitoring of certain folders](http://feedback.livereload.com/knowledgebase/articles/86239-os-x-fsevents-bug-may-prevent-monitoring-of-certai)
+
+### Windows paths
+
+webpack expects absolute paths for many config options. `__dirname + "/app/folder"` is wrong, because windows uses `\` as path separator. This breaks some stuff.
```

Comment:
@EugeneHlushko How about…

> Because webpack expects absolute paths for many config options such as `__dirname + "/app/folder"` the Windows `\` path separator can break some functionality.